static/mainapp/css/index.css

```diff
@@ -301,6 +304,15 @@ ol.carousel-indicators li.active {
 }
 /* media query*/
 
+@media screen and (max-width: 575px){
+  .col-lg-3:hover{
+    transform: scale(1.03);
+    transition: all 0.5s ease;
```

Comment:
Only transform needs to be handled by media query no need to repeat the remaining attributes